AlphaFilter/AlphaFilter.hpp

```diff
@@ -85,7 +87,11 @@ class AlphaFilter {
 	 *
 	 * @return retrieve the filtered result
 	 */
-	void update(const T &sample) { _filter_state = updateCalculation(sample); }
+	const T& update(const T &sample)
```

Comment:
```suggestion
	const T &update(const T &sample)
```
or adapt the getter just below depending on the desired style, I have not hard feelings on this one.

Reply:
We're doing it the other way for parameters:
e.g. `void setVelocityGains(const matrix::Vector3f &P, const matrix::Vector3f &I, const matrix::Vector3f &D);`
Or is that just me going against the stream?
No also here `Subscription &operator=(const Subscription &other)` 🤷‍♂️

Reply:
I'm happy with whatever can be done automatically (astyle or clang-format).